.ui/pgFavorites.pgx

```diff
@@ -50,10 +49,8 @@
 			"className": ".sf-listView .grow",
```

Comment:
we shouldn't be using user props inside page (for headerbar)

Reply:
done.